src/theme/DocItem/Layout/styles.module.css

```diff
@@ -0,0 +1,10 @@
+.docItemContainer header + *,
```

Comment:
Can we use `scss` for this file instead of `css`? We should be using that by default since it's used everywhere else.